unfoldingWord/translationCore
# Updated Naming Convention, Allow words to be moved after drop.

#### This pull request addresses:

Updated Naming Convention, Allow words to be moved after dropped. Can only move English words between the Greek words, nothing else.

#### How to test this pull request:

Must checkout branch of same name in tool and core.
Open the one test project that is compatible, and ensure words can be moved after moved from the word bank.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2742)
